sageresearch-app-sdk/src/main/java/org/sagebionetworks/research/sageresearch/dao/room/HistoryItemEntity.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * BSD 3-Clause License
+ *
+ * Copyright 2019  Sage Bionetworks. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ * 1.  Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * 2.  Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation and/or
+ * other materials provided with the distribution.
+ *
+ * 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission. No license is granted to the trademarks of
+ * the copyright holders even if such marks are included in this software.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.sagebionetworks.research.sageresearch.dao.room
+
+import androidx.room.Entity
+import org.threeten.bp.Instant
+
+@Entity(primaryKeys = arrayOf("reportId", "dateBucket", "dateTime"))
+data class HistoryItemEntity(
+
+
+        /**
+         * @property type of the history item
+         */
+        var type: String,
+
+        /**
+         * @property dataJson - json representation of this history item
+         */
+        var dataJson: String,
+
+        /**
+         * @property reportId of the report backing this history item
+         */
+        var reportId: String,
+
+        /**
+         * @property dateBucket day string in format: yyyy-MM-dd
+         */
+        var dateBucket: String,
+
+        /**
+         * @property dateTime of when the history item occurred
+         */
+        var dateTime: Instant
```

Comment:
Should we use ZonedDateTime? Depending on what Bridge sends or what iOS does.. we get bitten a lot when we store only UTC time or an OffsetDateTime and lose the Zone info